infra/src/Pos/Infra/Shutdown/Logic.hs

```diff
@@ -1,3 +1,5 @@
+{-# LANGUAGE RecordWildCards #-}
```

Comment:
I believe we're trying to get away from this extension :upside_down_face:

Reply:
Done!